extensions/spark/install_spark.sh

```diff
@@ -128,7 +128,6 @@ export SPARK_DAEMON_MEMORY=${SPARK_DAEMON_MEMORY}m
 export SPARK_WORKER_DIR=${SPARK_WORKDIR}
 export SPARK_LOCAL_DIRS=${SPARK_TMPDIR}
 export SPARK_LOG_DIR=${SPARK_LOG_DIR}
-export SPARK_CLASSPATH=\$SPARK_CLASSPATH:${LOCAL_GCS_JAR}
```

Comment:
Is this supposed to be in this PR? Last time we tried switching off SPARK_CLASSPATH, there was an issue using spark.driver.extraClassPath and spark.executor.extraClassPath because the master daemon wasn't picking up either classpath addition, so the master crashed on every job when trying to use GCS as the eventlog dir.

If it wasn't causing problems for Bigtable other than seeing the deprecation warning message, it's better to leave this in until the broader classpath issue is fixed for the GCS connector.